styles/tabs.less

```diff
@@ -215,16 +215,19 @@
 
 // Active pane marker --------------
 
-atom-pane.active .tab.active:before {
-  content: "";
-  position: absolute;
-  pointer-events: none;
-  z-index: 2;
-  top: 0;
-  left: -1px; // cover left border
-  bottom: 0;
-  width: 2px;
-  background: @accent-color;
+atom-pane-axis > atom-pane.active,
+atom-pane-container > atom-pane.pane {
```

Comment:
Nice! This would fix #204. 🎉 

Although, would it make more sense that the marker **only** shows with split panes? When there is just one pane, the marker wouldn't be needed.

![tabs](https://user-images.githubusercontent.com/378023/37901637-d2c5faa2-312c-11e8-9302-f44196e145ac.gif)

In that case the ☝️ above selector `atom-pane-container > atom-pane.pane`  could be removed.

Reply:
Personally, my opinion is that the active tab looks better with a marker anyway ...

The interface is much more alive ...

Especially when broken panels the marker is expected to be only in the active tab, the active panel.

![kapture 2018-03-26 at 16 17 52](https://user-images.githubusercontent.com/16165904/37908660-86263506-3111-11e8-9a5f-de07b736f5e9.gif)

But then again, this is just my opinion.

In any case, the decision to take you :)

Reply:
ok, let's merge this. We can still change it later if needed. Always seeing the marker might be good too, so you don't try to find it.

@9585999 Thanks for this PR. 🙇

Reply:
@simurai 
No problem. Thank you for this UI, as for me this is the best UI at the moment.